addons/pharma/functions/fnc_handleLimbIVComplications.sqf

```diff
@@ -0,0 +1,83 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Cplhardcore
+ * Handles advanced IV complications
+ *
+ * Arguments:
+ * 0: The Unit <OBJECT>
+ * 1: The Bodypart <String>
+ * 2: flowDifference (difference in fluid between the cap and the actual)<Number>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "LeftLeg", 2] call kat_pharma_fnc_handleLimbIVComplications
+ *
+ * Public: No
+ */
+
+ params ["_unit", "_bodypart", "_incomingFlowDifference"];
+
+ private _partIndex = ALL_BODY_PARTS find toLower _bodyPart;
+ private _IVarray = _patient getVariable [QGVAR(IV), [0,0,0,0,0,0]];
+ private _IVactual = _IVarray select _partIndex;
+
+ if (_bodypart == "Chest") then {
+    [_unit, 0.7] call ACEFUNC(medical_status,adjustPainLevel);
+ } else {
+    switch (true) do
+    {
+        case (_incomingFlowDifference < 2): {
+            if (random(100) < 10) then {
+                [_unit, 0.3] call ACEFUNC(medical_status,adjustPainLevel);
+                [objNull, _unit, _bodypart] call EFUNC(pharma,removeIV);
+            };
+        };
+        case ((_incomingFlowDifference < 4) && (_incomingFlowDifference <= 2)): {
+            if (random(100) < 25) then {
+                [_unit, 0.3] call ACEFUNC(medical_status,adjustPainLevel);
+                [objNull, _unit, _bodypart] call EFUNC(pharma,removeIV);
+            };
+
+            if (random(100) < 4) then {
+                _IVarray set [_partIndex, 9];
```

Comment:
Note this has the same issue with the IV values as in the pharma PR. The GUI does not know what to do with an _IVarray value of 9